praw/models/comment_forest.py

```diff
@@ -72,6 +72,12 @@ def __len__(self) -> int:
         """Return the number of top-level comments in the forest."""
         return len(self._comments)
 
+    def __eq__(self, other: _CommentForest):
+        """Compare two comment forest instances."""
+        if isinstance(other, CommentForest):
+            return self.list() == other.list()
+        return NotImplemented
```

Comment:
As elsewhere, this should be a call to `super().__eq__`.

Reply:
> I really don't see a practical and specific use-case in which a PRAW user would want to compare two `CommentForest`s in such a way. Specifically, it doesn't really make much sense to me that someone would ever have two different `CommentForest`s that are equal, since that would be redundant.

A plausible use case is monitoring a submission's comment for changes? Additions can be streamed, but maybe deletions would not be recognized. Although your point does stand true in that there's no real reason to check for deletions only. The whole point was to add in minor features that weren't too hard to code in that should just exist. However, there really is no point in making something that nobody will use.